src/Jsonize.Test/Program.cs

```diff
@@ -12,9 +12,11 @@ namespace Jsonize_Test
 {
     public class Program
     {
+        const string TEXTFILE = "WriteText.txt";
+        static string _html = null;
```

Comment:
It may seem strange but that's the way I do it even when C# is a high-level programming language. You'd be surprised what a variable can have as it's value when you let it unassigned (yes, I'm looking at you, C++ 😄).